_content/articles/religion-medicine-medieval-chinese-buddhism_salguero-p.md

```diff
@@ -0,0 +1,27 @@
+---
+title: "“Healing and/or Salvation? The Relationship Between Religion and Medicine in Medieval Chinese Buddhism"
+authors:
+  - salguero-p
+external_url: "https://www.multiple-secularities.de/media/workingpaper_4_salguero_final.pdf"
+drive_links:
+  - "https://drive.google.com/file/d/1FC7peNfI4CmBNVIeFcb0fVJ1s0-1EpZK/view?usp=sharing"
+course: medieval
+tags:
+  - health
+  - nibbana
+  - east-asian
+year: 2018
+month: apr
+journal: "Working paper series of the HCAS 'Multiple Secularities - Beyond the West, Beyond Modernities"
```

Comment:
🤔  Let's move this to `essays` and put `"Multiple Secularities Working Paper"` as a series name instead, with `number: 4`